data/json/items/gun/flammable.json

```diff
@@ -27,6 +27,25 @@
     ],
     "magazines": [ [ "flammable", [ "pressurized_tank" ] ] ]
   },
+  {
+    "id": "flamethrower_simple",
+    "copy-from": "flamethrower_base",
+    "looks_like": "m79",
+    "type": "GUN",
+    "name": "simple flamethrower",
+    "description": "A simple, home-made flamethrower.  While its capacity is not superb, it is more than capable of igniting terrain and monsters alike.",
+    "weight": "1900 g",
+    "volume": "1500 ml",
+    "price": 40000,
+    "to_hit": -1,
+    "bashing": 8,
+    "material": [ "steel", "plastic" ],
+    "range": 5,
+    "dispersion": 500,
+    "durability": 6,
+    "relative": { "reload": 2 },
+    "magazines": [ [ "flammable", [ "aux_pressurized_tank", "pressurized_tank" ] ] ]
```

Comment:
This shouldn't take detachable fuel tanks. It's just a fire extinguisher with a nozzle and a pilot light so it should have an integral magazine.